frontend/src/core/scaffold/views/scaffoldAssetView.js

```diff
@@ -284,6 +284,7 @@ define(function(require) {
 
             // Check if alternative attribute should be used
             if (alternativeAttribute) {
+                attributesToSave = [];
```

Comment:
It's declared at the top of the function, but given a value of `null` in certain cases.